src/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './components/index.js'
-export * from './CountlyMetrics.js'
+export * from './MetricsProvider.js'
```

Comment:
@whizzzkid how do you feel about this being a class rather than a set of functions? I ended up migrating the solution @djmcquillan put together to a class while debugging an issue that lead me to https://github.com/Countly/countly-sdk-web/issues/337, but functions may be easier to consume by react